src/webui/www/private/views/preferences.html

```diff
@@ -1294,24 +1294,14 @@
             $('use_peer_proxy_checkbox').setProperty('disabled', !isPeerProxyTypeSelected);
             $('proxy_only_for_torrents_checkbox').setProperty('disabled', !isPeerProxyTypeSelected);
 
-            if (isPeerProxyTypeSelected) {
-                const isPeerProxyTypeSocks5 = $('peer_proxy_type_select').getProperty('value') == "socks5";
-                $('peer_proxy_auth_checkbox').setProperty('disabled', !isPeerProxyTypeSocks5);
+            const isPeerProxyAuthenticatable = ($('peer_proxy_type_select').getProperty('value') == "socks5" || $('peer_proxy_type_select').getProperty('value') == "http");
```

Comment:
It wasn't possible to authenticate from WebUI for "http". I assume it's only a UI bug and the infrastructure exists (can't and won't test if proxies work) but if no one can confirm it then i should undo it.

Reply:
>and the infrastructure exists 

You can easily test it, see if the proxy settings in WebUI/GUI are kept in sync.

Reply:
Ok good, it seems to be in sync.